templates/pages/signup.jsx

```diff
@@ -87,7 +87,7 @@ var Signup = React.createClass({
           <IconText iconClass="agreement" textClass="eula">
             By signing up, I agree to Webmaker&lsquo;s <a tabIndex="5" href="//beta.webmaker.org/#/legal" className="underline">Terms of Service</a> and <a tabIndex="6" href="//www.mozilla.org/privacy/websites" className="underline">Privacy Policy</a>.
```

Comment:
Do we still want this to read as "Webmaker's" terms of service and privacy policy?

If we do want to change this wording, we might want to do that separately, because we'd have to talk to someone in legal.